xskillscore/tests/conftest.py

```diff
@@ -179,6 +179,19 @@ def weights_cos_lat(a):
     return xr.ones_like(a) * np.abs(np.cos(a.lat))
```

Comment:
I am unsure whether such weights are useful. shouldnt we only do weighting across what we specific in dim?

Reply:
at least this is what xarray does and it makes sense to me.